### ISSUE TITLE
turn "initialize.d" into a hook

### ISSUE THREAD

**Issue by gabriel-v** [liquidinvestigations/liquidinvestigations#256]
Make the `first-boot` hook only run steps needed to serve the `/welcome` page. Then, the welcome's `reconfigure` step runs `initialize.sh` that sets up the rest of the apps.

Then, split the `initialize.sh` script into two hooks: `initialize`, that only runs once after `/welcome` is submitted (and configures one-time things like the matrix configuration), and `configure` that runs every time the configuration changes (and takes care of apps being started, stopped and configured). The `configure` hook will also run on every subsequent boot.

@mgax please comment

**Comment by mgax** [comment]
Yep, I totally agree. The first-boot hook will need to run migrations for `liquid-core`, which should be pretty quick.
